netutils/mac.py

```diff
@@ -136,3 +136,29 @@ def mac_normalize(mac: str) -> str:
         if char in mac:
             mac = mac.replace(char, "")
     return mac
+
+
+@_valid_mac
+def get_oui(mac: str) -> t.Optional[str]:
```

Comment:
Why optional? should be `-> str` and then you can get rid of the # type ignore in the return line.

Reply:
As we discussed, I didn't have my `OUI_MAPPINGS` dict typed so it threw an error. Fixed now!